src/firebase/index.firebase.js

```diff
@@ -17,5 +17,6 @@ firebase.initializeApp({
 const db = firebase.firestore();
 const auth = firebase.auth();
 const stor = firebase.storage();
+const realdb = firebase.database();
```

Comment:
Is it possible to use cloud firestore instead of realtime db?

Reply:
We have 2 choices here, continue implementing using realtime db which will save our free firestore read limit or change to cloud firestore

Reply:
Yes

Reply:
Okay, I think it'd be better to implement realtime db, what do you think?

Reply:
Yes, it is possible.